### ISSUE TITLE
Static Hordes break the game on low city size

### ISSUE THREAD

**Issue by molkemon** [CleverRaven/Cataclysm-DDA#74414]
### Is your feature request related to a problem? Please describe.

As a returning player who hasn't played in a while I specifically tried to create a bit of an easier world by reducing city size in world gen options. Both me and I'd wager new players would generally think: smaller city = less zombies = easier. The tradeoff of course would be reduced resource availability. 

However, as it turns out, thanks to [this ](https://github.com/CleverRaven/Cataclysm-DDA/pull/69980) PR, this doesn't make the game easier, it makes it much, much harder, as now all the small hamlets with 2 houses have like 90 Zeds in them and it is impossible to enter these at all. 

Paradoxically, the larger cities are much safer and can at least partially be entered. 

This is completely counter to real world logic and extremely frustrating. These small hamlets should be completely empty. 

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/1060874/96e2f328-7675-45a2-8893-bc75b9c70d64)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/1060874/1f5d3597-5c7a-4a60-a5a6-23650462dfb8)

### Solution you would like.

First of all, static hordes should be made optional again in my opinion. There is 0 reason to force them upon the player. Keep them on by default, but let us turn them off.

Additonally, even when turned on, there shouldn't be 90 Zeds for every lone standing house in the middle of nowhere. Check for city size before spawning and just don't have any hordes whatsoever in towns that would not show up on real life maps.

**Comment by KHeket** [comment]
Have same situation here #72339

**Comment by kevingranade** [comment]
> First of all, static hordes should be made optional again

No, this doesn't actually fix anything, it's a red herring. The question is "how many zombies spawn in a town", which has nothing to do with implementation details like hordes.

This issue doesn't have enough information to even start on, how many should spawn? For what sized town? etc etc. This has none of that, just "there's more than I want".

**Comment by GuardianDll** [comment]
#58946 should be related

**Comment by kevingranade** [comment]
See my reply here as well: https://discourse.cataclysmdda.org/t/lore-rationalizing-why-there-are-too-many-zombies/29245/12?u=kevin.granade
tl;dr we're not going for "each town has the correct number of zombies in it to represent it's former population". That's a nice eventual goal, but right now towns are also the wrong size for that to make sense, and since zombies can't move around to intercept you, there need to be more of them essentially guarding the town from all directions. The goal is to match the amount of threat you encounter when entering a town to what it should be.  Now, I'm not going to argue we have a perfect number right now, just that "just dial it down to match the number of houses" is going to end up with the numbers being incredibly low and the threat from zombies being nonexistent. An adjustment needs to start from "ok how many should we have anyway" while taking the above things into account.
